### ISSUE TITLE
Propose to set default deck in New duel to random prebuilt?

### ISSUE THREAD

**Issue by lodici** [magarena/magarena#599]
Currently the default is any three color random deck but I almost always change to random prebuilt since these decks are more interesting with more thought in their construction.

**Comment by melvinzhang** [comment]
Makes sense, the random decks have a bit too much variance. The only cons is that are some prebuilt decks require the player to play in a particular way to win, which may not be that good for new players.

Are there particular decks there that are straightforward to play and doesn't need a detailed read of the deck list? Sometime ago I was looking into sample decks eg http://magic.wizards.com/en/articles/archive/arcana/magic-origins-sample-decks-2015-06-08 those are designed to be for introducing players to the game.

**Comment by melvinzhang** [comment]
Eventually we expect players to set the deck to something they enjoy, say particular colors they like to play or a prebuilt decks or their own creation etc. I think the importance of the default deck is for the first play experience. For someone trying out the program for the first time, they will likely not fiddle with the settings at all. They would want to quickly start a match to experience the game.

I would like to suggest picking out two specific decks (not random), that are straightforward to play (doesn't require specific knowledge of the deck list or combo, so players don't have to read the deck list) and gives a good challenge (AI plays well with the deck).

**Comment by lodici** [comment]
I am not really experienced enough in MTG to know whether a deck is beginner suitable. As a relative beginner myself I prefer to play white and green decks which to me seem to have simpler mechanics. I tend to read quite a lot into the deck description, I like the ones which explain how the deck should be played.

It would be great if we could identify or add new decks which are considered suitable for beginners.  Also if we could identify decks that we know the AI plays well. And a maybe a way of categorizing decks such as strength, speed, mechanic, etc.

**Comment by lodici** [comment]
> I would like to suggest picking out two specific decks (not random), that are straightforward to play (doesn't require specific knowledge of the deck list or combo, so players don't have to read the deck list) and gives a good challenge (AI plays well with the deck).

:+1:

**Comment by melvinzhang** [comment]
I like green/white as well for simple creature based strategies. The sample decks will be ideal as they were designed by the experts who looked into this problem. I'll have a check to see if any of them can be added as is, i.e. we support all the cards they use.

**Comment by lodici** [comment]
A very vague idea - I know there are lots of sites that cater for decks. I wonder if we could tap into that somehow. Like Firemind top decks but more general. Magarena would filter out only those decks which it can play.
